manifests/c/CyberCat/ADBAppControl/1.8.5/CyberCat.ADBAppControl.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CyberCat.ADBAppControl
+PackageVersion: 1.8.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ADBAppControl.exe
+  PortableCommandAlias: ADBAppControl
+Installers:
+- InstallerUrl: https://adbappcontrol.com/download/?lang=en&t=zip

```

Comment:
I'm not 100% sure how to proceed here, since the `InstallerUrl` is a superset of a reported URL. Maybe it's just the non-zip format that was reported?

Reply:
This is a portable zip which contains platform-tools (adb & fastboot) and a GUI executable.